console/src/components/Services/Data/TableModify/ModifyTable.js

```diff
@@ -210,6 +212,12 @@ class ModifyTable extends React.Component {
             <h4 className={styles.subheading_text}>Triggers</h4>
             <TriggerEditorList tableSchema={tableSchema} dispatch={dispatch} />
             <hr />
+            <h4 className={styles.subheading_text}>Check Constraints</h4>
+            <CheckConstraints
+              constraints={tableSchema.check_constraints}
```

Comment:
use util fn in `pgUtils` called `getTableCheckConstraints(table)` as defined in the earlier comment